src/BenchmarkDotNet/Diagnosers/PerfCollectProfiler.cs

```diff
@@ -196,6 +196,7 @@ private void EnsureSymbolsForNativeRuntime(DiagnoserActionParameters parameters)
         {
             string cliPath = parameters.BenchmarkCase.GetToolchain() switch
             {
+                CsProjClassicNetToolchain classic => classic.CustomDotNetCliPath,
```

Comment:
The `perfcollect` profiler plugin can only be used on Linux, and `CsProjClassicNetToolchain` is needed only on Windows, so this change is redundant. 
```suggestion
```